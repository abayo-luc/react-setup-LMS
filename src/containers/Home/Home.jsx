import React, { Component } from 'react';

export default class Home extends Component {
  state = {};

  render() {
    return (
      <div className="home">
        <h1>Hello World</h1>
      </div>
    );
  }
}
