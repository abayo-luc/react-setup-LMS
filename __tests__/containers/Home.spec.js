import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../../src/containers/Home/Home';

describe('Home container', () => {
  it('should match the snapshot', () => {
    const home = shallow(<Home />);
    expect(toJson(home)).toMatchSnapshot();
  });
});
