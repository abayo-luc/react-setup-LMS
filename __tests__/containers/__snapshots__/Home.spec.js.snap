// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home container should match the snapshot 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
